app/assets/javascripts/components/student/pages/lessons_page.es6.jsx

```diff
@@ -1,12 +1,35 @@
 class LessonsPage extends React.Component {
 
+  static get propTypes() {
+    return {
+      studentId: React.PropTypes.number.isRequired,
+    }
+  }
+
   constructor() {
     super();
     this.state = {
       filter: "upcoming",
+      lessons: null,
     };
   }
 
+  componentDidMount() {
+    this.fetchLessons();
+  }
+
+  fetchLessons() {
+    const {studentId} = this.props
+    const route = ApiConstants.students.upcoming_lessons(studentId);
```

Comment:
do u have any particular reasoning behind why you made this one a const?
i can see why you made it a const but can you make this a var for consistency for now, we can refactor that in the future possibly